### ISSUE TITLE
entry.sh hangs on "Lock file: /tmp/.X??-lock FOUND"

### ISSUE THREAD

**Issue by LewisW** [elgalu/docker-selenium#67]
When the container is rebooted, it hangs on "Lock file: /tmp/.X??-lock FOUND" which looking at the process is because of this command: `rm /tmp/.X10-lock`...

```
  119 ?        Ss     0:00 bash
  132 ?        R+     0:00  \_ ps afx
    1 ?        Ss+    0:00 /bin/sh -c "entry.sh"
    7 ?        S+     0:00 bash /usr/bin/entry.sh
   48 ?        S+     0:00  \_ bash /usr/bin/clear_x_locks.sh
   49 ?        S+     0:00      \_ rm /tmp/.X10-lock
```

I assume this is because the interactive message about `.X10-lock` being a write protected file is appearing and stopping everything. When I connect into the container and manually run `rm -f /tmp/.X10-lock` then `entry.sh` continues, and everything boots back up as normal.

**Comment by elgalu** [comment]
Thanks for the feedback Lewis 

So I borrowed this from https://github.com/SeleniumHQ/docker-selenium/pull/179 and not sure why I'm not able to reproduce the issue after `docker stop` && `docker start` again the container.

@garagepoort do you know what's wrong in your script / how can we fix this? will adding `-f ` to `rm` help?

**Comment by elgalu** [comment]
@LewisW if you find a solution could you send a pull request?

**Comment by garagepoort** [comment]
@elgalu I didn't have this problem as the file was not write protected for me. @LewisW could you test if changing the script with the force flag fixes it.

**Comment by garagepoort** [comment]
@LewisW I updated my pull request. If you could try again with the changes

**Comment by elgalu** [comment]
Should be fixed now starting version [2.53.0f](https://github.com/elgalu/docker-selenium/releases/tag/2.53.0f) through this changes: https://github.com/elgalu/docker-selenium/compare/2.53.0e...2.53.0f#diff-7a1b802aee1f12f8919ec3edbe581b36R5

**Comment by elgalu** [comment]
Also added a [test scenario](https://github.com/elgalu/docker-selenium/blob/2.53.0f/test/script_scenario_2#L7) to make sure tests run after `docker restart`
